### DISCUSSION TITLE
When will the 1.0.3 release?

### DISCUSSION THREAD

**Discussion by darkhog** [blackears/cyclopsLevelBuilder#156] [General]
I'm tired of the bugs in 1.0.2 that are supposedly already fixed for 1.0.3.

**Comment by blackears** [comment]
Sorry - I've kept pushing it back due to things happening in my personal life.  I hope to release it in the next couple of weeks.

**Reply by darkhog** [reply]
I see. In the meantime, how would I download what's already done for 1.0.3 directly from Github? Just clone the repo? Or do I have to do something extra?

**Reply by blackears** [reply]
You can clone the repo and then run the `create_addon_archive.py` script.  That will generate an archive with the most up to date code.
